index.html

```diff
@@ -30,16 +46,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id="box1" class="box">Box 1</div>
+                <div id="box2" class="box">Box 2</div>
+                <div id="box3" class="box">Box 3</div>
+                <div id="box4" class="box">Box 4</div>
+                <div id="box5" class="box">Box 5</div>
+                <div id="box6" class="box">Box 6</div>
+                <div id="box7" class="box">Box 7</div>
+                <div id="box8" class="box">Box 8</div>
+                <div id="box9" class="box">Box 9</div>
+                <div id="box10" class="box">Box 10</div>
             </div>
```

Comment:
while using ids are technically not wrong, i think best practices would recommend using sibling classes or nth child, we would want to use id for emphasis, so we would want to stay away from having 10 ids, an even more excessive way would be to use the important, but that is on the other side of the spectrum and definitely would want to avoid